rawpheno.install

```diff
@@ -369,11 +369,20 @@ function rawpheno_install() {
   $arr_cvid_cvname = array();
 
   foreach($cv as $cv_i => $cv_name) {
-    $tmp = tripal_insert_cv($cv_name, str_replace('_', ' ', ucfirst($cv_name)));
+    if (function_exists('chado_insert_cv')) {
+      $tmp = chado_insert_cv($cv_name, str_replace('_', ' ', ucfirst($cv_name)));
+    }
+    else {
+      $tmp = tripal_insert_cv($cv_name, str_replace('_', ' ', ucfirst($cv_name)));
+    }
+
+    if ($tmp) {
+      $arr_cvid_cvname[$cv_name] = $tmp->cv_id;
+    }
```

Comment:
Everytime you use an if like this, you should have an else with a tripal_report_error(). Otherwise, you may be masking the problem.